prac_07/convert_miles_km.kv

```diff
@@ -0,0 +1,29 @@
+BoxLayout:
+    orientation: 'vertical'
+    # text input and up and down arrows
+    BoxLayout:
+        orientation: 'horizontal'
+        TextInput:
+            id: miles
+            text: "0"
+#            on_text: app.convert_miles_to_kilometers(int(miles.text))
+        BoxLayout:
+            orientation: 'vertical'
+            Button:
+                text: "up"
+                on_press: app.handle_increment(miles.text, 1)
+            Button:
+                text: "down"
+                on_press: app.handle_increment(miles.text, -1)
+
+    # rest of body
+    Button:
+        text: "convert miles to kilometers"
+        on_press: app.convert_miles_to_kilometers(int(miles.text))
+    Label:
+        id: output_label
+        text: app.output_km
```

Comment:
As discussed in a comment below, 'miles' could be better renamed to 'input_number' in order to improve usability across multiple programs.